code/modules/clothing/clothing.dm

```diff
@@ -304,21 +313,19 @@
 	return TRUE
 
 /obj/item/clothing/under/proc/set_sensors(mob/user)
-	if(!user.Adjacent(src) || !ishuman(user) || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED))
-		return
```

Comment:
Will this allow you to open the TGUI menu at range?

Reply:
No, I have tested. You cannot open the TGUI menu if you are not adjacent.

Reply:
As said in discord, you can open the TGUI menu
![image](https://github.com/ParadiseSS13/Paradise/assets/108773801/475e72c7-af3f-4747-8d41-91e571297e94)